addons/l10n_br/data/template/account.tax.group-br.csv

```diff
@@ -73,3 +73,29 @@
 "tax_group_pis_deson_excl_goods","PIS Desoneração Excl.","base.br","account_template_202011003","account_template_102010802","PIS Desoneração Excl."
 "tax_group_pis_st_incl_goods","PIS ST Incl.","base.br","account_template_202011003","account_template_102010802","PIS ST Incl."
 "tax_group_pis_st_excl_goods","PIS ST Excl.","base.br","account_template_202011003","account_template_102010802","PIS ST Excl."
+"tax_group_aproxtrib_city_incl_services","Approximate Municipal Taxation Incl.","base.br","account_template_202011003","account_template_102010802","Tributação Municipal Aproximada Incl."
```

Comment:
There are usually very few tax groups, only one per rate.
What is the reason for more?

Reply:
@william-andre We are using Avalara to compute taxes, so we don't use the rate of the tax, but the tax amount given by Avalara instead.

That being said, Avalara had a few different taxes that are treated in different ways, those are the included and excluded. For the same tax, both options can apply depending on the case, and we want to split them on the invoice summary before the Total of the invoice.

If we only have one tax group for those 2 taxes, then the user won't know if the tax was included or excluded as it is coming from Avalara and it depends on the transaction details.

Not sure if this answers the question.
Thanks!
cc' @jorenvo

Reply:
@vbe-odoo Is this something that exists in other software as well, that difference between incl/excl?  Because, normally, we do not do that in tax groups. Also, why not reuse the one we already have for the goods?

Reply:
@jco-odoo they differentiate between included and excluded, as they are treated differently, even on other software, they have different amount fields on the invoice for each of them depending if they are included or excluded.

We do this also for goods, where we have different taxes and different tax groups, as we need to differentiate them also in the totals of the invoice, where we specify if they are included or excluded.

About on why not to reuse from goods, as far as I know, we are reusing the ones that we can, but for services we have others that are not the same as for goods. So we need to create the new ones, that are not there for services, indeed there are 3 taxes - with the excluded/included each so it will give you 6 taxes - that are repeated between goods and services, those are `aproxtribFed`, `cofins`, `pis`.

Here you will see some PIS, COFINS taxes but even those are different from goods, as you will see they have the RF together with the tax, this is because it is a different tax, it is treated as a WTH tax added to the invoice when apply depending on the transaction.

So, the main message is: 
- Ee are doing the same as with goods
- They are different taxes and treated differently between included/excluded
- We need the tax groups to differentiate them on the invoice as we do with goods (if not there is no easy way to see what they are when on the invoice in Odoo)
- We are reusing from goods what we can, but we need new taxes for services

cc' @jorenvo